boards/common/cc26xx_cc13xx/Makefile.include

```diff
@@ -13,5 +13,39 @@ PROGRAMMERS_SUPPORTED += openocd uniflash
 # embedded debugger of these launchpad boards.
 TTY_BOARD_FILTER := --model XDS110 --iface-num 0
 
-OPENOCD_CONFIG ?= $(RIOTBOARD)/common/cc26xx_cc13xx/dist/openocd.cfg
 UNIFLASH_CONFIG ?= $(RIOTBOARD)/common/cc26xx_cc13xx/dist
+
+OPENOCD_DEBUG_ADAPTER ?= xds110
+# Work around a bug in OpenOCD's handling of the ICEPick JTAG router
+OPENOCD_CMD_RESET_HALT ?= -c 'halt'
```

Comment:
I talked with OpenOCD developers. The issues with using just `halt` here is that any potentially registers interrupt handlers may still execute during `halt` - and an IRQ handler on a partially re-written program is unlikely to do any good.

A `reset halt` would reset the CPU (clearing the IRQ handlers) and then halt immediately - so that no IRQ handlers are registered.

I am pretty sure that the second reset (there is a `-c 'reset init'` also part of the flash script) is triggering a silicon bug in all CC26xx / CC13xx MCUs.

The integrated ICEPick JTAG router has the feature to keep the DAP in reset once a reset is detected (wait-in-reset). Likely this is a work around for that bug...